src/library_wasi.js

```diff
@@ -561,28 +561,20 @@ var WasiLibrary = {
   // random.h
 
   $initRandomFill: () => {
-    if (typeof crypto == 'object' && typeof crypto.getRandomValues == 'function') {
-#if SHARED_MEMORY
-      // like with most Web APIs, we can't use Web Crypto API directly on shared memory,
-      // so we need to create an intermediate buffer and copy it to the destination
-      return (view) => view.set(crypto.getRandomValues(new Uint8Array(view.byteLength)));
-#else
-      return (view) => crypto.getRandomValues(view);
-#endif
-    }
-
-#if ENVIRONMENT_MAY_BE_NODE
+#if ENVIRONMENT_MAY_BE_NODE && MIN_NODE_VERSION < 190000
     if (ENVIRONMENT_IS_NODE) {
-      return (view) => require('crypto').randomFillSync(view);
+      // Node v19 and above has globalThis.getRandomValues builtin
```

Comment:
This comment talks about `getRandomValues` but the code on the line below it is `randomFillSync`?

Reply:
The `crypto.getRandomValues` call is below.   It gets used when we don't need to support node < 19